api/AudioParam.json

```
@@ -624,10 +624,12 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "25"
+              "version_added": "25",
+              "notes": "Prior to Firefox 69, <code>value</code> did not take into accoutn scheduled or gradiated changes to the parameter's value; instead, only explicitly set values were returned."
```

Whoops, got a little typo here!

```suggestion
              "notes": "Prior to Firefox 69, <code>value</code> did not take into account scheduled or gradiated changes to the parameter's value; instead, only explicitly set values were returned."
```